test/test.oauth2.ts

```diff
@@ -950,11 +951,12 @@ it('should not retry requests with streaming data', done => {
   const scope = nock('http://example.com').post('/').reply(401);
   client.credentials = {
     access_token: 'initial-access-token',
-    refresh_token: 'refresh-token-placeholder'
+    refresh_token: 'refresh-token-placeholder',
+    expiry_date: (new Date()).getTime() + 500000
```

Comment:
I'm curious why this change was needed

Reply:
Without this, the test seemed to first try to fetch a new token, which would have required a different nock / more logic.